pkg/network/usm/sharedlibraries/ebpf.go

```diff
@@ -155,32 +401,51 @@ func (e *EbpfProgram) init(buf bytecode.AssetReader, options manager.Options) er
 		)
 	}
 
+	var enabledMsgs []string
+	for libset := range LibsetToLibSuffixes {
+		value := uint64(0)
+		if slices.Contains(libsets, libset) {
+			value = uint64(1)
+		}
+
+		constEd := manager.ConstantEditor{
+			Name:  fmt.Sprintf("%s_libset_enabled", string(libset)),
+			Value: uint64(1),
```

Comment:
What do you mean by "manual test"?

Reply:
I meant to add QA steps to manually check that if you enable gpu monitoring (only gpu monitoring) then we expect to see a certain programs/maps being loaded, and if only openssl monitoring is enabled then we expect to see only their related maps and programs being enabled, if both enabled -> then we expect to see everything is enabled, and if nothing is enabled then nothing is enabled

why manual test and not automated? because I'm not sure we have the infrastructure for that. We will need to test system-probe as a complete unit, and currently the tests are module focused.

Why is it beneficial to have the test? To ensure we're not causing troubles or we have bugs (like constant value of `1` as appears above)